installation/install.ps1

```diff
@@ -1,4 +1,9 @@
-﻿# This script installs all you need to use qcodes:
+# To run this installation script, right-click it and select run with PowerShell
+# To run it from ISE execute:
+# `Set-ExecutionPolicy -Scope Process -ExecutionPolicy Unrestricted`
+# to temporarily elevate the rights to executed foreign scripts
+
+# This script installs all you need to use qcodes:
```

Comment:
This is not completely true is it :) We are missing NI Visa?

Reply:
good point